tp_connected/tp_connected.py

```diff
@@ -10,7 +10,7 @@
 import base64
 from .tp_link_encryption import tp_link_encryption
 
-TIMEOUT = 3
+TIMEOUT = 300
```

Comment:
From the description I understand that we need to increase the timeout just for the reconnection, increasing the Timeout for all the operations from 3 seconds to 300 seems a lot. Can we have separate timeout values for the reconnection and the operations like `getParm`?

Reply:
@andtos90 , I've changed the login related stuff to have a big timeout (5 minutes) and left the send sms requests with the original timeout.
Here is a log of reconnect, for to see the timing:
```
 2021-08-15 17:00:23 INFO (MainThread) [homeassistant.helpers.script.websocket_api_script] websocket_api script: Running websocket_api script
2021-08-15 17:00:23 INFO (MainThread) [homeassistant.helpers.script.websocket_api_script] websocket_api script: Executing step call service
2021-08-15 17:00:23 DEBUG (MainThread) [tp_connected.tp_connected] Send to XXX-XXXXXXX via http://192.168.1.1/ len=9
2021-08-15 17:00:23 DEBUG (MainThread) [tp_connected.tp_connected] Sent message with status 500
2021-08-15 17:00:23 DEBUG (MainThread) [tp_connected.tp_connected] autologin
2021-08-15 17:00:23 INFO (MainThread) [tp_connected.tp_connected] http://192.168.1.1/cgi/getParm
2021-08-15 17:00:23 DEBUG (MainThread) [tp_connected.tp_connected] ee: 010001 nn: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX



2021-08-15 17:00:40 DEBUG (MainThread) [tp_connected.tp_connected] Encrypted username: XXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXX


2021-08-15 17:00:56 DEBUG (MainThread) [tp_connected.tp_connected] Encrypted password: XXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXX
2021-08-15 17:00:57 INFO (MainThread) [tp_connected.tp_connected] http://192.168.1.1/cgi/login
2021-08-15 17:00:58 DEBUG (MainThread) [tp_connected.tp_connected] Session id: XXXXXXXXXXXXXXXxx
2021-08-15 17:00:58 INFO (MainThread) [tp_connected.tp_connected] Token url http://192.168.1.1/
2021-08-15 17:00:58 DEBUG (MainThread) [tp_connected.tp_connected] Valid token request
2021-08-15 17:00:58 DEBUG (MainThread) [tp_connected.tp_connected] Token id: XXXXXXXXXXXXXXXXXXXxx
2021-08-15 17:00:58 DEBUG (MainThread) [tp_connected.tp_connected] Send to XXX-XXXXXXX via http://192.168.1.1/ len=9
2021-08-15 17:00:58 DEBUG (MainThread) [tp_connected.tp_connected] Sent message with status 200


```